apis/python/src/tiledb/vector_search/utils.py

```diff
@@ -4,6 +4,14 @@
 
 import tiledb
 
+def add_to_group(group, uri, name):
+    '''
+    Adds an object to a group. Automatically infers whether to use a relative path or absolute path.
```

Comment:
Please add a note about why and cross-ref the sc-<id> ticket number here so that we can trace back to be able to remove this eventually.

Reply:
Thanks, done.